spirekey/src/components/Card/PasskeyCard.tsx

```diff
@@ -34,7 +34,7 @@ export default function PasskeyCard({
             onSuccessfulAnimationEnd?.();
           }}
         >
-          {!fingerprintAnimationDone && (
+          {!isSuccessful && (
```

Comment:
These should be as they were, as in that way the fingerprint is switched for the logo AFTER the animation is done (see line 49 of this file)

Reply:
it didn't work, this is a work around to get it to work (might be due to refactoring)